spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/Util.scala

```diff
@@ -179,10 +183,23 @@ object Util {
     var i = 0
     while (i < localWeightBias.length) {
       if (localWeightBias(i) != null) {
-        localWeightBias(i).set(broadcastWeightBias(i))
+        clearAndSet(localWeightBias(i), broadcastWeightBias(i))
       }
       i += 1
     }
+
+    def clearAndSet(old: Tensor[T], other: Tensor[T]): Unit = {
+      if (old.getTensorType == QuantizedType && other.getTensorType == QuantizedType) {
```

Comment:
do this in the set method of quantized tensor?

Reply:
It can't do `release` in the quentized tensor set method. Because if we have tensor A, B, C, than we do `A.set(B)` and `A.set(C)`. At the end, the `A` and `B` will be all released. This is not we want because we may want to reserve tensor `B`. So we only release the tensor carefully when needed.